com.usebruno.Bruno.metainfo.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.usebruno.Bruno</id>
+
+  <name>Bruno</name>
+  <summary>Fast and Git-Friendly Opensource API Client</summary>
+  <developer_name>Bruno</developer_name>
+  <url type="homepage">https://www.usebruno.com/</url>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
```

Comment:
the description say it's open source. which is it?

Reply:
While majority of features are opensource, we plan to ship some paid offerings. Hence I kept the license as propietary.
Github: https://github.com/usebruno/bruno
Paid offering: https://www.usebruno.com/pricing

If needed, I can remove the `Opensource` word from the description